config/config.go

```diff
@@ -132,10 +132,21 @@ func (d *Data) GetTemplateFilesDir() (path string) {
 	case filepath.IsAbs(ConfigFilePath):
 		d.Template.absDirectory = filepath.Join(filepath.Dir(ConfigFilePath), d.TemplatesDir, d.Template.Name)
 		return d.Template.absDirectory
-
 	// No absolute dir provided. Let's build one
 	default:
 		d.Template.absDirectory = filepath.Join(ProcWorkingDir, filepath.Dir(ConfigFilePath), d.TemplatesDir, d.Template.Name)
+		if filepath.IsAbs(d.Template.absDirectory) {
```

Comment:
The fact that this can't just be another case in the switch is a smell to me. Not a problem with what you're doing, but we should make a note to look at how we handle path building.